src/index.ts

```diff
@@ -152,6 +152,11 @@ function _pushable<PushType, ValueType, ReturnType> (getNext: getNext<PushType,
       return pushable
     }
 
+    // @ts-expect-error `byteLength` is not declared on PushType
+    if (options?.objectMode !== true && value.byteLength == null) {
```

Comment:
value can also be not an object which would throw confusing error?

Reply:
The idea is it's consistent with node's concept of [object mode streams](https://nodejs.org/dist/latest-v16.x/docs/api/stream.html#object-mode) so I don't \*think\* this is a problem, but I could be wrong. Did you find it suprising?

Reply:
Nah, my bad!